RELEASE

```diff
@@ -13,6 +13,7 @@ appstream-util appdata-to-news ../data/org.freedesktop.fwupd.metainfo.xml > NEWS
 2. Update translations:
 
 ninja-build fwupd-pot
+ninja-build uefi-update-quirk
```

Comment:
You should probably mention this as an extra step "Refresh quirks"

Reply:
Fixed